content/en/architecture/security/contingency-plan.en.md

```diff
@@ -0,0 +1,49 @@
+
+## Contingency Overview
+
+The primary objective of Aurora's contingency plan is to ensure the **continuous 24/7 operation of the Aurora platform**, while minimizing any disruption to client workloads. Aurora is committed to maintaining uninterrupted service outside of scheduled maintenance windows, ensuring clients can rely on the platform for consistent availability.
+
+Aurora achieves this objective without having to compromise or deteriorate Aurora's security & safeguards through the following ways:
+
+- **Architectural resiliency**: Core components of the Aurora platform are deployed in high availability (HA) configurations to ensure system uptime and fault tolerance.
+- **Client support**: Dedicated support is available during **business hours** to assist clients with issues related to their workloads.
+- **Operational readiness**: A comprehensive set of <gcds-link href="{{< relref "/team/standard-operating-procedures" >}}">Standard Operating Procedures (SOPs)</gcds-link> and alert
+<gcds-link href="{{< relref "/team/monitoring-alerts/" >}}">runbooks</gcds-link> is maintained to enable quick incident troubleshooting & resolution by the Aurora team.
+
+>Business hours is defined as 9:00 AM - 5:00 PM EST, Monday to Friday.
+
+This plan helps Aurora meet the targets for the target resolution times, as listed below.
+
+### Target Resolution Times
+
+| **Severity Level** | Description                                         | **Target Resolution Time (90% of the time)** |
+|--------------------|-----------------------------------------------------|----------------------------------------------|
+| Critical           | Critical issue currently affecting all users, including system unavailability and major data integrity or security issues.                                                     | 4 hours                                      |
+| High               | Loss of functionality of non-critical components or a warning of imminent critical component failure.                                                    | 8 hours                                      |
+| Medium             | Significant & persistent performance issues or some bug affecting some but not all users.                                                    | 2 business days                              |
+| Low                | Minor issues with minimal impact on system functionality or user workloads.                                                    | 6 business days                              |
+
+> The **Target Resolution Time** is the average time taken to restore a service if an incident occurs. When a situation cannot be resolved within the target time, notifications will be provided to the IT partners.
+
+Priority is given to issues based on severity level & if the application affected is in the CBAS.
```

Comment:
I am thinking this part gets into support for clients workloads & I am thinking the split between the two (platform commitment & workload support commitment) should be clear in the documentation. Do you think my understand is correct?